portal-impl/src/com/liferay/portal/servlet/filters/aggregate/AggregateFilter.java

```diff
@@ -546,9 +534,7 @@ else if (minifierType.equals("js")) {
 										finalContent);
 								}
 								else {
-									minifiedContent =
-										MinifierUtil.minifyJavaScript(
-											finalResourcePath, finalContent);
+									minifiedContent = finalContent;
```

Comment:
minor nit, this can be removed and just initialize the variable in line 530 as follows:
```
   String minifiedContent = finalContent;
```

not sure about Brian's taste about this one though

Reply:
I'm going to change it :+1:  Thx.